bin/oio-blob-mover

```diff
@@ -16,12 +16,34 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import argparse
+
 from oio.blob.mover import BlobMover
 from oio.common.daemon import run_daemon
-from oio.common.configuration import parse_options
-from optparse import OptionParser
+
+def make_arg_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('config', help="Configuration file")
```

Comment:
Still using a configuration file for compatibility with previous versions is a good think, but now there are command line parameters, requiring it is a bit weird, I think.